### PR TITLE
major updates on bas ui 4 s4hc tutorial group 

### PR DESCRIPTION
**Pull request by Liebherr**
Hi Arianna,
thanks for reviewing again

https://developers-qa-blue.wcms-nonprod.c.eu-de-2.cloud.sap/tutorials/abap-custom-ui-bas-connect-s4hc.html
https://developers-qa-blue.wcms-nonprod.c.eu-de-2.cloud.sap/tutorials/abap-custom-ui-bas-develop-s4hc.html
https://developers-qa-blue.wcms-nonprod.c.eu-de-2.cloud.sap/tutorials/abap-custom-ui-generate-tile.html

Regards
Ulrike

### CODE REVIEW

File: tutorials/abap-custom-ui-bas-connect-s4hc/abap-custom-ui-bas-connect-s4hc.md [hunk 1]
```diff
@@ -33,76 +34,77 @@ Alternatively, see [Integrating SAP Business Application Studio](https://help.sa
...
+    |  Field Name     | Value          | Remark
+    |  :------------- | :------------- | :-------------
+    |  HTML5.DynamicDestination           | **`true`**   |&nbsp;
+    |  HTML5.Timeout           | **`60000`**   | value stated in milliseconds. 60000 equals 1 minute. Required as deployment needs longer than the standard of 30 seconds.
+    |  `WebIDEEnabled`    | **`true`**   |&nbsp;
+    |  `WebIDEUsage`          | **`odata_abap,dev_abap`**   |&nbsp;
+    |  `nameIDFormat`     | **`urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress`**  | Required in case your subaccount sends mail address as SAML Name ID for authentication ( **Subject Name Identifier** in Identity Authentication tenant), although SAP S/4HANA Cloud tenant expects user login by default. **That is the case with a trial Account.** This also requires the mail address to be maintained for SAP S/4HANA Cloud tenant business users.
```
Comment (AriannaMussoBarcucci) [inline tutorials/abap-custom-ui-bas-connect-s4hc/abap-custom-ui-bas-connect-s4hc.md:88]: typo: e-mail

File: tutorials/abap-custom-ui-bas-connect-s4hc/abap-custom-ui-bas-connect-s4hc.md [hunk 2]
```diff
@@ -134,27 +136,27 @@ This is how you have to configure the communication system that represents the S
...
-    <!-- border -->![Copy and Paste Provider Name ](s4hc-cs-fill-saml-provider-name.png)
+    ![Copy and Paste Provider Name ](s4hc-cs-fill-saml-provider-name.png)
 
 - Choose **Save**.
 
-The connection is now set up and you can use of the custom business object OData services of the SAP S/4HANA Cloud tenant in SAP Business Application Studio.  
+The connection is now set up and you can make use of the custom business object OData services of the SAP S/4HANA Cloud tenant in SAP Business Application Studio.  
 
 
 ### Test yourself
```
Comment (AriannaMussoBarcucci) [inline tutorials/abap-custom-ui-bas-connect-s4hc/abap-custom-ui-bas-connect-s4hc.md:162]: Test yourself section is empty

File: tutorials/abap-custom-ui-bas-develop-s4hc/abap-custom-ui-bas-develop-s4hc.md [hunk 3]
```diff
@@ -23,29 +23,29 @@ primary_tag: topic>abap-extensibility
...
+    ![Enter trial Subaccount](btp-enter-trial-subaccount.png)
 
 3.  In the navigation pane expand the **Services** section.
 
 4.  Select **Instances and Subscriptions**.
 
 5.  Click the link or the icon at the SAP Business Application Studio Subscription.
 
-<!-- border -->![Open SAP Business Application Studio from Subaccount](btp-open-bas.png)
+![Open SAP Business Application Studio from Subaccount](btp-open-bas.png)
```
Comment (AriannaMussoBarcucci) [inline tutorials/abap-custom-ui-bas-develop-s4hc/abap-custom-ui-bas-develop-s4hc.md:48]: similar to the previous tutorial: I personally found the numbering on the screenshot a bit confusing. If I were reading the tutorial for the first time I would assume I screenshot with steps 1 and 2 is somehow missing. I would suggest to keep the numbering in the screenshot sequential (1,2,3) and maybe add it after every step, i.e:
"In the navigation pane expand the Services section (1)" and so on.

File: tutorials/abap-custom-ui-bas-develop-s4hc/abap-custom-ui-bas-develop-s4hc.md [hunk 4]
```diff
@@ -125,11 +125,11 @@ In this step, you set project attributes and choose to add further optional conf
 
 In this step, you define where you want the UI project to be deployed to as a runnable application.
 
-<!-- border -->![UI Wizard: Configure Deployment Settings](bas-appgen-deploy-config.png)
+![UI Wizard: Configure Deployment Settings](bas-appgen-deploy-config.png)
 
 1. Leave the default `ABAP` as **target** platform and as **Destination** the one you have created earlier (see [Connect SAP Business Application Studio and SAP S/4HANA Cloud tenant](abap-custom-ui-bas-connect-s4hc)).
 
-2. Enter a **Name** that represents the UI5 ABAP repository the application will be deployed to. Example: `YY1_BONUSPLAN`. This repository name will be visible as your **Custom UI App ID** in your SAP S/4HANA Cloud tenant.
+2. Enter the **SAPUI5 ABAP Repository** Name that will be created and the application be deployed to. Example: `YY1_BONUSPLAN`. This repository name will be visible as your **Custom UI App ID** in your SAP S/4HANA Cloud tenant.
```
Comment (AriannaMussoBarcucci) [inline tutorials/abap-custom-ui-bas-develop-s4hc/abap-custom-ui-bas-develop-s4hc.md:132]: Enter a name for the **SAPUI5 ABAP Repository**. This is the repository that will be created and where the application will be deployed.

File: tutorials/abap-custom-ui-bas-connect-s4hc/abap-custom-ui-bas-connect-s4hc.md [hunk 5]
```diff
@@ -5,19 +5,20 @@ time: 20
...
+
 ## You will learn
 - How to create an HTTP destination on SAP Business Technology Platform with SAML assertion authentication to an SAP S/4HANA Cloud tenant
 - How to create a communication system for an SAP Business Application Studio subaccount in an S/4HANA ABAP tenant
 
 ## Intro
-To follow this tutorial, you can either use a subaccount in your trial account on SAP BTP or you can use a subaccount in a customer account. However, for the customer account subaccount, you have to do the following:
+To follow this tutorial, you can either use a subaccount in your trial account on SAP BTP or you can use a subaccount in a customer account. However, for the customer account subaccount, you have to do the following additionally beforehand:
 
   - Set up mutual trust between the SAP BTP subaccount and the Identity Provider, see
```
[the remark this answers is not in the document]
Comment (Liebherr) [reply tutorials/abap-custom-ui-bas-connect-s4hc/abap-custom-ui-bas-connect-s4hc.md:23]: will be replaced by completely matching tutorial soon

File: tutorials/abap-custom-ui-bas-develop-s4hc/abap-custom-ui-bas-develop-s4hc.md (hunk 3 above)
Comment (Liebherr) [reply tutorials/abap-custom-ui-bas-develop-s4hc/abap-custom-ui-bas-develop-s4hc.md:48]: left the numbering but put screenshot up to first step it shows

File: tutorials/abap-custom-ui-bas-connect-s4hc/abap-custom-ui-bas-connect-s4hc.md [hunk 6]
```diff
@@ -33,76 +34,77 @@ Alternatively, see [Integrating SAP Business Application Studio](https://help.sa
...
-    <!-- border -->![Enter trial Subaccount](btp-enter-trial-subaccount.png)
+    ![Enter trial Subaccount](btp-enter-trial-subaccount.png)
 
 3.  In the navigation pane expand the **Connectivity** section.
 
-4.  Select **Destinations**.
-
-5.  Select **New Destination**.
-
-<!-- border -->![Get to New Destination](btp-destination-new.png)
```
[the remark this answers is not in the document]
Comment (Liebherr) [reply tutorials/abap-custom-ui-bas-connect-s4hc/abap-custom-ui-bas-connect-s4hc.md:58]: previous steps have own screenshot
yes, numbers are bullet numbers
i think having a screenshot per bullet point is too much if all can be shown in one
i think having different numbers in screenhot than at bullet point and mentioning it there makes it confusing too
-> i'll leave it as it is

**Pull request merged**